app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id]) 
+    @challenge = Challenge.find(@project.challenge_id)
+    @count = @project.contestants.count
+  end
```

Comment:
Remember we want to keep our controllers slim. So we only want what we need in here. So we really only want/need line 3. Line 4 - in your view you can reach through `@project` to get to its challenge. 
Line 5 - it is not the controllers job to do data logic. That is our models job. So you would want to create a method in your `Project` model that gets the count. Then you can call that method in your view on `@project`. So something like:
```ruby
def num_contestants
  contestants.count
end
```
Then in your view you can call `@project.num_contestants`